L1Trigger/L1THGCal/interface/backend/HGCalHistoSeedingImpl.h

```diff
@@ -159,7 +159,10 @@ class HGCalHistoSeedingImpl {
   Navigator navigator_;
 
   static constexpr unsigned neighbour_weights_size_ = 9;
-  static constexpr double kROverZMin_ = 0.076;
+  // HGCAL value
+  //  static constexpr double kROverZMin_ = 0.076;
+  // HGCAL nose
+  static constexpr double kROverZMin_ = 0.02;  // r=30 cm at z=1000 cm
```

Comment:
The number of bins has to be changed accordingly to keep the same bin size as before

Reply:
ok, will adjust and check with the other suggestion

Reply:
@jbsauvan Do you have a description on how change the binning.
doesn't seems a 1:1 scaling with the range

Reply:
Actually it is. The bin size is constant in R/Z. The number of bins can be increased by the ratio of the new vs old boundaries (rounded to the closest integer). R/Z bins are given in `nBins1_` (there are generic names because we can have different types of binnings). The binning should both be modified in the source default value and in the python config.

Reply:
done, now we have 46 bins